src/Gui/PreferencePages/DlgSettingsEditor.ui

```diff
@@ -162,6 +162,9 @@
         <property name="toolTip">
          <string>Tabulator raster (how many spaces)</string>
         </property>
+        <property name="suffix">
+         <string> spaces</string>
```

Comment:
Is it possible to add translator comments in a *.ui file? I bet in some translations that space gets lost.